agent/run.sh

```diff
@@ -215,7 +215,7 @@ setup_state()
     cp -f /usr/bin/r /.r/r || true
 
     for m in $MODULES; do
-        modprobe $m >/dev/null 2>&1 || true
+        nsenter -m -t 1 modprobe $m >/dev/null 2>&1 || true
```

Comment:
should really be `/sbin/modprobe` or whatever it's guaranteed to be on a RHEL host.  The path resolution will happen in the container, which is bad, so you want to put an absolute file.